plugin/nix-exprs/sbtix.nix

```diff
@@ -103,13 +103,25 @@ let
         );
 
 in rec {
-    mkRepo = name: artifacts: runCommand name {}
+    mkRepo = name: artifacts: localBuildsRepo: runCommand name {}
         (let
             parentDirs = filePath:
                 concatStringsSep "/" (init (splitString "/" filePath));
             linkArtifact = outputPath: urlAttrs:
                 let
-                  artifact = fetchurl urlAttrs;
+                  localpath = p:
```

Comment:
Is this line useful to anything? It doesn't seem like a valid Nix syntax to me.

Reply:
Oops, indeed that was a leftover from a previous refactor, not sure how that slipped through. Updated and tested, should be good now.